app/components/Notifications/Notification.jsx

```diff
@@ -216,21 +216,37 @@ const Notification = ({
         <FormattedMessage id={NOTIFICATIONS_DATA[type].id} values={values} />
       </Span>
       <div className="d-flex align-items-center justify-content-between">
-        <a
-          href={isAnotherCommItem ? `${process.env.APP_LOCATION}${href}` : href}
-          className="d-flex align-items-center"
-        >
-          <IconMd
-            icon={NOTIFICATIONS_DATA[type].src}
-            color={
-              !isCommunityMod && tipNotification ? BORDER_WARNING_LIGHT : null
-            }
-            specialStyles={
-              isCommunityMod && tipNotification && styles.coinsIconStyles
-            }
-          />
-          <span>{data.title}</span>
-        </a>
+        {isAnotherCommItem && (
+          <a
+            href={`${process.env.APP_LOCATION}${href}`}
+            className="d-flex align-items-center"
+          >
+            <IconMd
```

Comment:
This code is duplicated. Just change `<a>` and `<Link>` element based on `isAnotherCommItem`

Reply:
fixed